src/it/java/com/google/checkstyle/test/chapter5naming/rule527localvariablenames/LocalVariableNamesTest.java

```diff
@@ -19,14 +19,16 @@
 
 package com.google.checkstyle.test.chapter5naming.rule527localvariablenames;
 
-import java.util.Map;
-
 import org.junit.jupiter.api.Test;
 
 import com.google.checkstyle.test.base.AbstractGoogleModuleTestSupport;
-import com.puppycrawl.tools.checkstyle.api.Configuration;
 
-public class PatternVariableNameTest extends AbstractGoogleModuleTestSupport {
+public class LocalVariableNamesTest extends AbstractGoogleModuleTestSupport {
+
```

Comment:
I didn't combined input files of `LocalVariableName` & `PatternVariableName` modules into one because the `PatternVariableName` had only one input file which was in resources-noncompileable directory so I thought it'd be better if we don't combine the input files of these modules otherwise the would need to put all the input code into resources-noncompileable input file.

Reply:
> I didn't combined input files of LocalVariableName & PatternVariableName modules into one

combining all to one was never required.
we should be able always ready to deal with multiple Input files.
As we come to whole config execution and google formatter usage for Inputs that has no violation, we will have to use multiple Input files.